products/memgraph/v0.51.0/how_to_guides/use-and-implement-query-modules.md

```diff
@@ -0,0 +1,322 @@
+## How to Use and Implement Query Modules?
+
+Memgraph supports extending the query language with procedures written by a
+user. These procedures are compiled into modules, which can then be loaded on
+startup.
```

Comment:
`Memgraph supports extending the query language with user written procedures.` ?
`These procedures are grouped into modules, which can then be loaded on startup.` ?